README.md

```diff
@@ -1,7 +1,35 @@
 [![Build Status](https://travis-ci.org/CODAIT/MAX-Object-Detector-Web-App.svg?branch=master)](https://travis-ci.org/CODAIT/MAX-Object-Detector-Web-App)
 
-# Object Detector Web App
-A web app wrapping the [MAX Object Detector](https://github.com/IBM/MAX-Object-Detector) model output
+# Create a web app to visually interact with objects detected using machine learning
+
+In this Code Pattern we will use one of the models from the
+[Model Asset eXchange (MAX)](https://developer.ibm.com/code/exchanges/models/),
+an exchange where developers can find and experiment with open source deep learning models.
+Specifically we will be using the [Object Detector](https://github.com/IBM/MAX-Object-Detector)
+to create a web application that will recognize objects in an image and allow the user to
+filter the objects based on their detected label and prediction accuracy. The web application
+provides an interactive user interface backed by a lightweight Node.js server using Express.
+The server hosts a client-side web UI and relays API calls to the model from the web UI to a REST
+end point for the model. The Web UI takes in an image and sends it to the model REST endpoint
+via the server and displays the detected objects on the UI. The model's REST endpoint is set up
+using the docker image provided on MAX. The Web UI displays the detected objects in an image
```

Comment:
Capitalize "Docker ..."

Reply:
docker should be lowercase when referring to a "docker image" it's only uppercase when referring to the company or application

Reply:
it's consistent across MAX, and Docker docs don't actual use docker mostly because it's all about docker and so the adjective is unnecessary, but I did see it in a few places on the Docker site outside the docs at some point, and IBM Cloud docs seem to follow what I'm doing as well.